src/modules/requirements/QuestCompletedRequirement.ts

```diff
@@ -0,0 +1,22 @@
+import { AchievementOption } from '../GameConstants';
+
+import Requirement from './Requirement';
+
+export default class QuestLineCompletedRequirement extends Requirement {
```

Comment:
This class seems to have the same name as the below class, some possible alternatives:
```suggestion
export default class QuestLineStepCompletedRequirement extends Requirement {
```
```suggestion
export default class QuestLineCompletedStepRequirement extends Requirement {
```
```suggestion
export default class QuestLineQuestCompletedRequirement extends Requirement {
```
```suggestion
export default class QuestLineCompletedQuestRequirement extends Requirement {
```

Reply:
I was planning to call it "QuestCompletedRequirement", but my copy/paste-skills failed me...
But i like "QuestLineStepCompletedRequirement" a lot better, so i choose that